Gemfile

```diff
@@ -15,11 +15,12 @@ group :test do
   gem 'license_finder', '~> 6.1.2'
   gem 'parallel', '= 1.13.0'
   gem 'parallel_tests', '~> 2.24.0'
-  gem 'parser', '~> 2.7.1'
+  # gem 'parser', '~> 2.7.1'
```

Comment:
This gem commented out, is it an oversight?

Reply:
This can actually be removed. I'll do that now.